### PR TITLE
Document new -XX:[+|-]ShowUnmountedThreadStacks option

### LINKED ISSUE #1241: Document new -XX:[+|-]ShowUnmountedThreadStacks option
**Issue or pull request number:**
*https://github.com/eclipse-openj9/openj9/pull/18218*

**Overview:**
*Option controls if unmounted virtual thread stacks are included in java core file.*

**Release target:**
*Eclipse OpenJ9 0.44.0*

**Applies to the following JDK versions:**
*21 and later*

**Applies to the following platforms:**
*All platforms*

**For new command line options:**

- Option name and syntax: *-XX:[+|-]ShowUnmountedThreadStacks*
- Default value: *disabled*
- Purpose: Allow unmounted virtual thread stack to be include in java core file.

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
https://github.com/eclipse-openj9/openj9-docs/issues/1241

Added a new topic for the new option, -XX:[+|-]ShowUnmountedThreadStacks

Closes #1241
Signed-off-by: Sreekala Gopakumar <sreekala.gopakumar@ibm.com>

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@fengxue-IS - I have updated the document. Please check and confirm. Thanks!

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by fengxue-IS** [comment]
just a note the unmounted threads also include carrier thread if there is a virtual thread mounted to it hence unmounting the carrier itself

**Comment by Sreekala-Gopakumar** [comment]
> just a note the unmounted threads also include carrier thread if there is a virtual thread mounted to it hence unmounting the carrier itself

Does that mean the stack of any unmounted carrier thread is also not included in the core file?

**Comment by fengxue-IS** [comment]
> Does that mean the stack of any unmounted carrier thread is also not included in the core file?

yes

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@fengxue-IS @pshipton - I have made the suggested changes. Please check and confirm. Thanks!

File: docs/xxshowunmountedthreadstacks.md
Comment (pshipton) [inline docs/xxshowunmountedthreadstacks.md:2]: The copyright end date is incorrect.

Comment (pshipton) [reply docs/xxshowunmountedthreadstacks.md:2]: There are two other topics using a 2022 end date.
xxcodecachetotalmaxrampercentage.md
xxcontinuationcache.md

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have updated the documents. Please check and confirm. Thanks!

File: .vscode/settings.json
Comment (pshipton) [inline .vscode/settings.json:3]: Is this meant to be included in the commit?

Comment (Sreekala-Gopakumar) [reply .vscode/settings.json:3]: I am not sure, Peter. This seems to have changed after the WCA extension was added to VS code as a part of the WCA@IBM initiative. If this doesn't have an impact on the OpenJ9 builds, then I guess it can be included in the commit.

Comment (pshipton) [reply .vscode/settings.json:3]: Pls remove it from the commit. You can update the https://github.com/eclipse-openj9/openj9-docs/blob/master/.gitignore file to ignore it in the future.

Comment (Sreekala-Gopakumar) [reply .vscode/settings.json:3]: ok

**Comment by pshipton** [comment]
Waiting on Jack's review before merging.

**Review by fengxue-IS — APPROVED** [review]
lgtm

**Review by pshipton — APPROVED** [review]

**Pull request merged**
